Scroller/useThemeScroller.js

```diff
@@ -103,6 +86,9 @@ const getFocusableBodyProps = (scrollContainerRef, contentId, isScrollbarVisible
 		}
 	};
 
+	// Initial filter setting
+	setNavigableFilter({filterTarget: 'body'});
```

Comment:
I'm afraid this be calling whenever Scroller got re-rendered. Shouldn't we put this in `useEffect` with [] dependency?